marpa/lib/Guacamole.pm

```diff
@@ -804,6 +804,11 @@ OpKeywordSocketExpr           ::= OpKeywordSocket Expression OpComma Expression
 
 OpKeywordSocketpairExpr       ::= OpKeywordSocketpair Expression OpComma Expression OpComma Expression OpComma Expression OpComma Expression
 
+OpKeywordSortExpr             ::= OpKeywordSort Block Expression
+                                | OpKeywordSort NonBraceExpression Expression
```

Comment:
Docs are misleading here, not all expressions can be used as SUBNAME. I did not check the code, but it seems only barewords and variables work in this position. From reading the docs I'd expect this to also work, but it doesn't (fortunately): `sort "foo" 1, 2`

You already have `VarScalar` case, and I'm not sure we want to allow barewords here?

Reply:
Only `VarScalar`, not bareword.

Reply:
So just removing this line?

Reply:
Yes